orbit/initializer/dlt.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
Maybe we can nest this module inside `model` so it'd live in `orbit/model/initializer` or even just `orbit/model/model_init.py` or perhaps even in `utils`. Not sure if it should have its own entire module

Reply:
So a module is not an option since callalbe is not allowed inside pickle unless it is an object.  Meanwhile, I am open to put it under model_init instead of a new folder but I think it is minor?